### PR TITLE
chore(docs): Document integration with NodeJS

### PR DESCRIPTION
**Pull request by bastiaanv**
In this PR, I added documentation about how you can utilise babel to integrate React Email into your NodeJS application. I also updated the imports to match the `export default` props

### CODE REVIEW

File: apps/docs/integrations/aws-ses.mdx [hunk 1]
```diff
@@ -45,10 +45,10 @@ export default function Email(props) {
 
 Import the email template you just built, convert into a HTML string, and use the AWS SES SDK to send it.
 
-```js index.tsx
+```js index.ts
```
Comment (bukinoshita) [inline apps/docs/integrations/aws-ses.mdx:48]: It needs to be `tsx` otherwise it will complain about the Email component.

File: apps/docs/integrations/nodemailer.mdx [hunk 2]
```diff
@@ -45,10 +45,10 @@ export default function Email(props) {
 
 Import the email template you just built, convert into a HTML string, and use the Nodemailer SDK to send it.
 
-```js index.tsx
+```js index.ts
```
Comment (bukinoshita) [inline apps/docs/integrations/nodemailer.mdx:48]: It needs to be `tsx` otherwise it will complain about the Email component.

File: apps/docs/integrations/aws-ses.mdx [hunk 3]
```diff
@@ -45,10 +45,10 @@ export default function Email(props) {
...
 import { render } from '@react-email/render';
 import AWS from 'aws-sdk';
-import { Email } from './email';
+import Email from './email';
```
Comment (bukinoshita) [inline apps/docs/integrations/aws-ses.mdx:51]: Why the change here?

Comment (bastiaanv) [reply apps/docs/integrations/aws-ses.mdx:53]: Because you are exporting the function as default, you can only import it this way. https://stackoverflow.com/a/33307487

File: apps/docs/integrations/aws-ses.mdx (hunk 1 above)
Comment (bastiaanv) [reply apps/docs/integrations/aws-ses.mdx]: @bukinoshita You are right. I forgot about that. Because of the babel package, you can now use the component as a function. I updated the docs, you now have a React and a NodeJS variant

File: apps/docs/integrations/aws-ses.mdx (hunk 3 above)
Comment (bukinoshita) [reply apps/docs/integrations/aws-ses.mdx:53]: This is more of user's preference than a rule. Use can export the component either way.

Comment (bastiaanv) [reply apps/docs/integrations/aws-ses.mdx]: Reverted changes

**Comment by bukinoshita** [comment]
@bastiaanv Sorry for the delay, but thanks for the PR ❤️
